frontend_app_share_folder_advanced/test/.mocharc.js

```diff
@@ -6,7 +6,9 @@ module.exports = {
     'regenerator-runtime/runtime',
     '@babel/register',
     'ignore-styles',
-    'test/setup'
+    'test/setup',
+    // INFO - CH - 2020-06-05 - Isomorphic-fetch provide the fetch API because Node does not contain it, and unit test run in Node context
```

Comment:
CH?

Reply:
haha, I guess he just copast the comment I made